projects/plugins/jetpack/extensions/blocks/map/component/mapkit.js

```diff
@@ -80,11 +106,25 @@ const MapkitComponent = forwardRef(
 						onMapLoaded={ onMapLoaded }
 					/>
 				) : null }
+
+				{ /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */ }
```

Comment:
Noting that I'm just focussed on mouse click events - this PR doesn't introduce other accessibility issues but I did notice existing ones that should get fixed later: https://github.com/Automattic/jetpack/issues/38986